tests/testthat/helper-orderly.R

```diff
@@ -1,6 +1,6 @@
 options(outpack.schema_validate =
           requireNamespace("jsonvalidate", quietly = TRUE) &&
-          packageVersion("jsonvalidate") >= "1.4.0",
+          utils::packageVersion("jsonvalidate") >= "1.4.0",
```

Comment:
For some reason I've been getting errors on this line (even before this PR). Possibly CI never caught it because it doesn't have `jsonvalidate` installed?

Reply:
ci should have jsonlite installed I think, there's a Remotes line in DESCRIPTION that will get the dev version; what error were you getting?

Reply:
Okay this turns out to just be because of my setup. I was doing a `pkgload::load_all()` inside of my RProfile to get orderly2 to load automatically, but at the point when RProfile is run, the `utils` package isn't attached yet.

I'll remove it.